src/resources/views/crud/fields/select_and_order.blade.php

```diff
@@ -95,6 +101,8 @@
 {{-- FIELD JS - will be loaded in the after_scripts section --}}
 @push('crud_fields_scripts')
 @loadOnce('packages/jquery-ui-dist/jquery-ui.min.js')
+@loadOnce('//cdnjs.cloudflare.com/ajax/libs/jqueryui-touch-punch/0.2.2/jquery.ui.touch-punch.min.js')
```

Comment:
We are not moving back to CDNs in this release, so I think it's better to use touch-punch from a local file instead of CDN. Can you please add it to Backpack's `package.json` and load it from the published file? I see it exists on NPM - https://www.npmjs.com/package/jquery-ui-touch-punch

Reply:
Done! It's now an npm dependency 👌